dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/WebContainer.java

```diff
@@ -191,7 +191,7 @@ public Object run() {
 
     }
 
-    protected static boolean decodePlusSign = true; //default to not decoding the plus sign per URL Spec
+    protected boolean decodePlusSign = true; // Default true, but activate() will set this from WCCustomProperties.DECODE_URL_PLUS_SIGN
```

Comment:
Can you explain why this isn't static now?

Reply:
Also the whole getDecodePlusSign method just seems out of place to me here, do we know why this is here rather than just referencing WCCustomProperties for a static value in places where getDecodePlusSign would have usually been called? Looking at the OL code I only saw one instance in `DynamicVirtualHost`?

Reply:
I changed back to an earlier revision which removes getDecodePlusSign() and variable decodePlusSign.